src/main/java/bio/terra/app/configuration/ApplicationConfiguration.java

```diff
@@ -397,6 +399,11 @@ public ExecutorService performanceThreadpool() {
         new LinkedBlockingQueue<>(getMaxPerformanceThreadQueueSize()));
   }
 
+  @Bean
+  public RestTemplate restTemplate(RestTemplateBuilder builder) {
+    return builder.build();
+  }
+
```

Comment:
I don't really understand why we seemingly need this RestTemplate bean. It appears we reference a rest template without the bean w/ some of our other referenced services.

Reply:
Ok! I think actually referencing the RestTemplate bean as you suggested cleaned this up and I'm no longer concerned about it! Thank you!